test/javascript/node-project.test.ts

```diff
@@ -662,6 +662,24 @@ test("enabling dependabot does not overturn mergify: false", () => {
   expect(snapshot).toHaveProperty([".github/dependabot.yml"]);
 });
 
+test("enabling renovatebot does not overturn mergify: false", () => {
+  // WHEN
+  const project = new TestNodeProject({
+    renovatebot: true,
+    projenVersion: "0.50.0",
+    minNodeVersion: "14.0.0",
+    mergify: false,
+  });
+
+  // THEN
+  const snapshot = synthSnapshot(project);
+  // Note: brackets important, they prevent "." in filenames to be interpreted
+  //       as JSON object path delimiters.
+  expect(snapshot).not.toHaveProperty([".mergify.yml"]);
+  expect(snapshot).toHaveProperty(["renovate.json5"]);
+  expect(snapshot["renovate.json5"]).toMatchSnapshot();
```

Comment:
Can we also add an assertion on the list of ignored dependencies, by parsing the json5 file? e.g.

```suggestion
  expect(json5.parse(snapshot["renovate.json5"]).ignoreDeps).toEqual(...);
  expect(snapshot["renovate.json5"]).toMatchSnapshot();
```

Reply:
Added checking of ignores in the test